aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -139,6 +156,59 @@ module aptos_framework::coin {
         *allow_upgrades = allowed;
     }
 
+    //
+    //  Aggregatable coin functions
+    //
+
+    /// Creates a new aggregatable coin with value overflowing on `limit`. Note that this function can
+    /// only be called by Aptos Framework (0x1) account for now becuase of `create_aggregator`.
+    public(friend) fun initialize_aggregatable_coin<CoinType>(aptos_framework: &signer): AggregatableCoin<CoinType> {
```

Comment:
Do we need these functions here or can this be done directly in transaction_fee (with added friendship between transaction_fee and aggregatable coin)?

Reply:
`initialize_aggregatable_coin` and `is_aggregatable_coin_zero` can be moved to transaction_fee, but `drain_aggregatable_coin`, `merge_aggregatable_coin` and `collect_from_into_aggregatable_coin` have to stay in coin module to allow calling Coin constructor